be/src/olap/storage_policy.cpp

```diff
@@ -48,7 +48,7 @@ Status get_remote_file_system(int64_t storage_policy_id,
         return Status::InternalError("could not find resource, resouce_id={}",
                                      storage_policy->resource_id);
     }
-    DCHECK(atol((*fs)->id().c_str()) == storage_policy->resource_id);
+    // DCHECK(atol((*fs)->id().c_str()) == storage_policy->resource_id);
```

Comment:
This DCHECK macro is unnecessary because the field id is not initialized in the constructor of the class HdfsFileSystem,and neither the derived class nor the base class don't expose the interface of modifying such field.

Reply:
So the when the cooldown() invokes the DCHCK macro here, the whole BE node will fail because the id filed of fs pointer is empty.